OpenEdXMobile/res/drawable/edx_enroll_button.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="false">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/edx_brand_gray_base" />
```

Comment:
<img width="505" alt="screen shot 2017-06-08 at 09 45 36" src="https://user-images.githubusercontent.com/3858265/26917619-4377c732-4c2f-11e7-96f9-67450ba7a19a.png">
What about this color? @farhan @lizcohen